doc/source/release/1.14.0-notes.rst

```diff
@@ -106,25 +194,433 @@ Deprecated features
 Backwards incompatible changes
 ******************************
 
```

Comment:
```suggestion
- Many `scipy.stats` functions now produce a standardized warning message when an
  input sample is too small (e.g. zero size). Previously, these functions may have
  raised an error, emitted one or more less informative warnings, or emitted no 
  warnings. In most cases, returned results are unchanged; the most common
  correct result in such cases is NaN.
```

Reply:
Text looks good to me, but not sure if this needs to go under "Backwards incompatible changes" - I don't think we promise stability of warning messages (especially not the kind of spurious ones that PR removes), so this could/should IMO be just a regular news item in the stats section.

Reply:
I interpreted https://github.com/scipy/scipy/issues/19805#issuecomment-2100106063 to mean that the message should go in this section. I am happy either way, but I am concerned that if I move it, it would be contrary to that suggestoin.